src/lib/gas-refund-api.ts

```diff
@@ -185,22 +202,42 @@ export class GasRefundApi {
       },
     });
 
-    const pendingRefundableAmount = (await GasRefundTransaction.sum<
-      string,
-      GasRefundTransaction
-    >('refundedAmountPSP', {
-      where: {
-        address,
-        epoch: {
-          [Sequelize.Op.gt]: latestEpochRefunded,
+    const rawPendingData: PendingRefundRawData[] =
```

Comment:
I'm curious why did you go from model interface to query? Is it because models don't support grouping?

Reply:
smthg like that or because I was to lazy to research haha